application.py

```diff
@@ -613,10 +617,14 @@ def create_graph(output_cellset, includeisolated=False, graphname="cellgraph"):
     """
     global e
     # Graph using EM2 or use metadata coordinates
-    if application.config["GRAPH_EM2"]:
+    if application.config["GRAPH_METHOD"] == "em2":
```

Comment:
if this becomes a common pattern, you can use an enum to store the name of the graph method along with the function call.

Reply:
Good idea. I'll add that into my epic on being able to plug in various methods for the computations